### PR TITLE
NH-3606 - A method to create a child stateless session

### PR DESCRIPTION
**Pull request by rjperes**
#910, https://nhibernate.jira.com/browse/NH-3606

### CODE REVIEW

**Comment by fredericDelaporte** [comment]
May I rebase this PR?

**Comment by rjperes** [comment]
OK with me.

File: src/NHibernate/Linq/LinqExtensionMethods.cs [hunk 1]
```diff
@@ -13,6 +13,14 @@ namespace NHibernate.Linq
 {
 	public static class LinqExtensionMethods
 	{
+		public static IQueryable<T> AsStateless<T>(this IQueryable<T> query)
```
Comment (hazzik) [inline src/NHibernate/Linq/LinqExtensionMethods.cs]: I don't like this extensions. It needs to be implemented as a separate feature.

**Comment by hazzik** [comment]
@fredericDelaporte is this still required?

**Comment by fredericDelaporte** [comment]
This NH-3606 feature is not covered by the new "session sharing some parts of another" way of opening sessions, having replaced child sessions. Hibernate Java does not have it either, so having realigned on Hibernate for session opening has not brought this feature in.

I guess it could still be useful and added to current master, but this would require another PR. (Or rewriting most changes of this one.)

**Comment by fredericDelaporte** [comment]
Replaced by #1520

**Pull request closed without merging**